ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml

```diff
@@ -4760,4 +4760,234 @@ HWRWISOVPM:
   - heating_return_water_isolation_valve_percentage_command
   - heating_return_water_isolation_valve_percentage_sensor
   implements:
-  - MONITORING
\ No newline at end of file
+  - MONITORING
+  
+EPM:
+  description: "Basic Electrical parameter monitoring."
+  is_abstract: true
+  uses:
+  - power_status
+  - voltage_sensor
+  - current_sensor
+  - power_sensor
+  - energy_accumulator
+  implements:
+  - MONITORING"
+  
+VOADC2X:
+  description: "Variable outside air damper control and monitoring, where there are two separate, equal sets of dampers that operate in conjunction."
+  is_abstract: true
+  uses:
+  - outside_air_damper_percentage_sensor_1
+  - outside_air_damper_percentage_sensor_2
+  - outside_air_damper_percentage_command_1
+  - outside_air_damper_percentage_command_2
+  implements:
+  - CONTROL  
+  
+RAIDS3X:
+  description: "Return air isolation damper control and monitoring, where there are three separate, equal sets of dampers that operate in conjunction."
+  is_abstract: true
+  uses:
+  - return_air_isolation_damper_status_1
+  - return_air_isolation_damper_status_2
+  - return_air_isolation_damper_status_3  
+  implements:
+  - MONITORING
+
+RAIDS2X:
+  description: "Return air isolation damper control and monitoring, where there are two separate, equal sets of dampers that operate in conjunction."
+  is_abstract: true
+  uses:
+  - return_air_isolation_damper_status_1
+  - return_air_isolation_damper_status_2 
+  implements:
+  - MONITORING
+
+RAIDS4X:
+  description: "Return air isolation damper control and monitoring, where there are four separate, equal sets of dampers that operate in conjunction."
+  is_abstract: true
+  uses:
+  - return_air_isolation_damper_status_1
+  - return_air_isolation_damper_status_2
+  - return_air_isolation_damper_status_3
+  - return_air_isolation_damper_status_4  
+  implements:
+  - MONITORING
+
+SFRMM:
+  description: "Supply fan run mode monitoring."
+  is_abstract: true
+  uses:
+  - supply_fan_run_mode
+  implements:
+  - MONITORING
+
+SRC:
+  description: "Very basic run scheduling command. "
+  is_abstract: true
+  uses:
+  - schedule_run_command
+  implements:
+  - CONTROL
+
+IDSSC:
```

Comment:
maybe also follow the convention you set for RAIDC and call this SAIDC

Reply:
IDSSC is changed to SAIDC